### PR TITLE
Add Vue useful comments

### PR DESCRIPTION
**Pull request by rebekris**
Ad useful comments on App.vue and NoteEdit.vue

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -14,8 +15,11 @@ class NoteClass {
 export default {
    name: 'app',
 
+//components: Component registry to use
```
Comment (1VinceP) [inline src/App.vue]: Please indent and add a space after //

File: src/App.vue [hunk 2]
```diff
@@ -48,11 +53,18 @@ export default {
...
+   /*
+   It is invoked after an instance has been destroyed. All directives for VUE instances
+   have been unbound, listeners have been removed.
+   */
    destroyed() {
       clearInterval(this.interval);
    },
 
+// methods:It allows to declare methods that could be used in our project
+
```
Comment (1VinceP) [inline src/App.vue]: Indent and remove line break

File: src/App.vue [hunk 3]
```diff
@@ -48,11 +53,18 @@ export default {
...
    methods: {
+
```
Comment (1VinceP) [inline src/App.vue]: Remove line break

File: src/components/Note.vue [hunk 4]
```diff
@@ -1,7 +1,12 @@
 <script>
+
+//export default{} allows you to use the component within the project
+
 export default {
+// Name: Componet name
    name: 'note',
 
+//Props: Add properties to the object
```
Comment (1VinceP) [inline src/components/Note.vue]: Indent, add space after //

File: src/components/Note.vue [hunk 5]
```diff
@@ -1,7 +1,12 @@
 <script>
+
+//export default{} allows you to use the component within the project
+
```
Comment (1VinceP) [inline src/components/Note.vue]: Remove surrounding line breaks, add space after //

File: src/App.vue [hunk 6]
```diff
@@ -14,8 +15,11 @@ class NoteClass {
 export default {
    name: 'app',
 
+//components: Component registry to use
    components: { Note, NoteEdit },
 
+   //data: It is the container of variables, which may be accessible from the view => data: {}
+   //When working with components, it changes to the function format => data(){}
```
Comment (1VinceP) [inline src/App.vue]: Add space after //

**Review by 1VinceP — CHANGES_REQUESTED** [review]
Looks good, just have some small style changes I'd like you to make

**Comment by rebekris** [comment]
thanks, I done!.. and doing PR

**Review by 1VinceP — APPROVED** [review]

**Pull request merged**